lib/fluent/plugin_helper/cert_option.rb

```diff
@@ -45,6 +45,11 @@ def cert_option_create_context(version, insecure, ciphers, conf)
         if extra && !extra.empty?
           ctx.extra_chain_cert = extra
         end
+        if conf.cert_verifier
+          verifier = File.exist?(conf.cert_verifier) ? File.read(conf.cert_verifier) : conf.cert_verifier
+          ctx.verify_callback = self.instance_eval(verifier)
```

Comment:
Is it better to change the receiver of `instance_eval`?

If `File.exist?(conf.cert_verifier` returns  `true`, I think it's better to pass filename to the second argument of `instance_eval` so that users can debug easily.

https://ruby-doc.org/core-2.7.0/BasicObject.html#method-i-instance_eval

Reply:
> Is it better to change the receiver of instance_eval?

Which object?

Reply:
How about creating a new class like CerVerifierSandbox ?